From ab55ea181247df477aea24f6c4e555989f881b98 Mon Sep 17 00:00:00 2001
From: Li Jun <jun.li@freescale.com>
Date: Wed, 20 May 2015 14:40:59 +0800
Subject: [PATCH] MLK-10930-2 arm: imx7d: add phy-clkgate-delay-us property

In i.MX7D, we should wait 400us after put PHY into low power mode command
(PHCD), then turn off PHY clock to make sure PHY can be suspended correctly.

Signed-off-by: Li Jun <jun.li@freescale.com>
---
 arch/arm/boot/dts/imx7d.dtsi | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/boot/dts/imx7d.dtsi b/arch/arm/boot/dts/imx7d.dtsi
index d7614ca..273199d 100644
--- a/arch/arm/boot/dts/imx7d.dtsi
+++ b/arch/arm/boot/dts/imx7d.dtsi
@@ -1095,6 +1095,7 @@
 				clocks = <&clks IMX7D_USB_CTRL_CLK>;
 				fsl,usbphy = <&usbphy_nop1>;
 				fsl,usbmisc = <&usbmisc1 0>;
+				phy-clkgate-delay-us = <400>;
 				status = "disabled";
 			};
 
@@ -1105,6 +1106,7 @@
 				clocks = <&clks IMX7D_USB_CTRL_CLK>;
 				fsl,usbphy = <&usbphy_nop2>;
 				fsl,usbmisc = <&usbmisc2 0>;
+				phy-clkgate-delay-us = <400>;
 				status = "disabled";
 			};
 
@@ -1117,6 +1119,7 @@
 				fsl,usbmisc = <&usbmisc3 0>;
 				phy_type = "hsic";
 				dr_mode = "host";
+				phy-clkgate-delay-us = <400>;
 				status = "disabled";
 			};
 
-- 
1.8.0

